wiki/People/Users_with_unique_titles/en.md

```diff
@@ -0,0 +1,320 @@
+# Users with unique titles
+
+User titles are typically associated with [user groups](/wiki/People/The_Team#user-groups), but the users listed here were given their titles for other reasons. Most of the titles were rewards for contribution or achievement, with the remaining being tokens of iconic or funny osu! history.
+
+## By category, with reasons
+
+### World Cup winners
+
+One of the prizes given to first-place winners of [official World Cups](/wiki/Tournaments#official-world-cups) is a user title that lasts for one year.
+
+Winners of OWC 2018 with the **osu! Champion** user title:
+
+- ![][flag_US] [Apraxia](https://osu.ppy.sh/users/4194445)
+- ![][flag_US] [DigitalHypno](https://osu.ppy.sh/users/4384207)
+- ![][flag_US] [idke](https://osu.ppy.sh/users/4650315)
+- ![][flag_US] [Monko2k](https://osu.ppy.sh/users/4852013)
+- ![][flag_US] [t[-\_-t]](https://osu.ppy.sh/users/2644828)
+- ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689)
+- ![][flag_US] [Vaxei](https://osu.ppy.sh/users/4787150)
+
+Winners of TWC 2019 with the **osu!taiko Champion** user title:
+
+- ![][flag_JP] [ekumea1123](https://osu.ppy.sh/users/9119501)
+- ![][flag_JP] [iceOC](https://osu.ppy.sh/users/5482401)
+- ![][flag_JP] [isobe](https://osu.ppy.sh/users/8047347)
+- ![][flag_JP] [uone](https://osu.ppy.sh/users/5321719)
+- ![][flag_JP] [v2fax](https://osu.ppy.sh/users/7739269)
+
+Winners of CWC 2019 with the **osu!catch Champion** user title:
+
+- ![][flag_KR] [[224]Hyperw7](https://osu.ppy.sh/users/4158549)
+- ![][flag_KR] [Berea](https://osu.ppy.sh/users/3657951)
+- ![][flag_KR] [Motion](https://osu.ppy.sh/users/3885626)
+- ![][flag_KR] [qwhj1027](https://osu.ppy.sh/users/7547506)
+- ![][flag_KR] [R-18](https://osu.ppy.sh/users/4637369)
+
+Winners of MWC 4K 2019 with the **osu!mania Champion** user title:
+
+- ![][flag_KR] [blueBloody](https://osu.ppy.sh/users/8046839)
+- ![][flag_KR] [Kalkai](https://osu.ppy.sh/users/10790649)
+- ![][flag_KR] [SuddenDeath](https://osu.ppy.sh/users/6699923)
+- ![][flag_KR] [Transcendence](https://osu.ppy.sh/users/903155)
+- ![][flag_KR] [wonder5193](https://osu.ppy.sh/users/8474029)
+- ![][flag_KR] [Yeoul](https://osu.ppy.sh/users/4698728)
+
+Winners of MWC 7K 2019 with the **osu!mania Champion** user title:
+
+- ![][flag_KR] [Estonians](https://osu.ppy.sh/users/7014697)
+- ![][flag_KR] [Jakads](https://osu.ppy.sh/users/259972)
+- ![][flag_KR] [NKEYBOARD](https://osu.ppy.sh/users/10132430)
+- ![][flag_KR] [wonder5193](https://osu.ppy.sh/users/8474029)
```

Comment:
I have mixed feelings about having to update the article 5 times a year. Maybe delete the installments and players and leave something like `Winners of the {TYPE} World Cup receive the **osu!{TYPE} Champion** user title` or something like this

Reply:
this article is supposed to list all of the people, though. world cups already need other wiki things to be updated regularly